### DISCUSSION TITLE
How to create an HTTPS server in localhost? | Leo's blog - Leo's coding journey

### DISCUSSION THREAD

**Discussion by giscus** [hthai2201/haiht.dev#1] [Q&A]
# How to create an HTTPS server in localhost? | Leo's blog - Leo's coding journey

The simplest way to set up an HTTPS server in Nodejs application when developing locally

https://haiht-dev.vercel.app/blog/use-https-in-local-development#comments-container

**Comment by hthai2201** [comment]
thank
